frontend/src/audio.html

```diff
@@ -30,15 +30,31 @@
     <div class="mt-1 flex flex-col justify-between md:flex-row">
       <div class="flex flex-wrap content-between align-baseline text-xs child-div:mr-1 child-div:mt-1 child-div:border-r child-div:border-gray-300 child-div:pr-1 last:child-div:border-0">
         <div>#{{ audio.objectID }}</div>
-        <div>{{ audio.dateForHumans }}</div>
-        <div>{{ audio.location }}</div>
-        <div>{{ audio.category }}</div>
+        {% if audio.dateForHumans != blank %}
+          <div>
+            {{ audio.dateForHumans }}
+            {% if audio.dateUncertain != false %}
```

Comment:
`!= false` actually means `== true`. I think it's more straightforward. Moreover, you can use truthiness check here, I think.